nixos/modules/services/backup/borgbackup.nix

```diff
@@ -102,6 +102,16 @@ let
       inherit (cfg) startAt;
```

Comment:
would prefer that non-`systemd.timers` timer config goes away if we also use `systemd.timers` config. it's easier to understand when all information for a thing is in one place.

Reply:
you mean moving this line down to the timer part?

Reply:
yup, all it does now anyway is add an `OnCalendar` entry to the timer.